graphs/possible_bipartition.py

```diff
@@ -1,5 +1,3 @@
-# Can be used for BFS
-from collections import deque 
 
 def possible_bipartition(dislikes):
```

Comment:
👍  This works, time/space complexity?

It does seem a little overly complicated.  Could you simplify it with a single stack (instead of a stack for each node in the graph), and using a pair of sets, one for each dog pen?